src/frontend/Player/createPlyrPlayer.spec.ts

```diff
@@ -291,18 +291,4 @@ describe('createPlyrPlayer', () => {
     expect(mockCreateHlsPlayer).toHaveBeenCalled();
```

Comment:
in your commit message:
- does not exists anymore > does not exist anymore
- - in all the front application > in all the frontend application